src/messenger/email/email-service.ts

```
@@ -43,6 +44,7 @@ export class EmailService implements MessengerService {
     itemStorage?: BlDocumentStorage<Item>,
     inputPostOffice?: PostOffice,
   ) {
+    sgMail.setApiKey(process.env["SENDGRID_API_KEY"] ?? "SG.");
```

Dude please throw if not set

---

I implemented assertEnv for all environment variables. The current approach was jalla 🤠 
https://github.com/boklisten/bl-api/pull/533/commits/fdeae7b9617d09408247ba873b9f70939cfe7004